### DISCUSSION TITLE
Class "string" not found

### DISCUSSION THREAD

**Discussion by marlonbasten** [livewire/livewire#7574] [7. Bugs (archived)]
### Livewire version

v3.3.1

### Laravel version

v10.38.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

I upgraded Livewire 3.3.0 to 3.3.1 today and got a weird error on my component. It worked with Livewire 3.3.0. The Laravel framework version doesn't seem to matter.

<img width="1663" alt="image" src="https://github.com/livewire/livewire/assets/31259698/52cd7ce7-497e-44ab-9448-bf7a0a6536a1">

When I remove the code that created the error according to the Exception, it errors somewhere else.

### Repository or Wirebox URL

3.3.1 not available in wirebox

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by sanis** [comment]
His happens for me as well, and looks like it's coming from recent change: 

https://github.com/livewire/livewire/commit/7d4089f0d8364fe2de5c7872724b897c4ad0b183

**Comment by marlonbasten** [comment]
@calebporzio It still isn't fixed in Livewire 3.3.2. Maybe revert this change so things don't break for others

**Reply by joshhanley** [reply]
@marlonbasten we'd rather fix the issue. I just need someone to share some code so I can replicate the issue, then I can get it fixed up 🙂

**Comment by fhlarif** [comment]
Same thing happened to me today as well since I just upgraded to Livewire 3.0.2. Had to remove `string` type in public variables that uses `#[Url]` as a workaround for now.

**Reply by joshhanley** [reply]
@fhlarif can you clarify what you mean, as I'm struggling to replicate the issue? Do you mean if I do

```php
#[Url]
public string $test;
```

That should break? Is there any steps I need to take to trigger the error?

**Reply by fhlarif** [reply]
```php
 #[Url(as: 'pt', history: true)]
    public string $patientSearch = '';
```

**Comment by joshhanley** [comment]
Can someone share me a small component class and view that demonstrates this issue so I can have a look and get it fixed? Thanks!

**Reply by fhlarif** [reply]
I've made a simple example that showcase the bug. [String Class Bug](https://github.com/fhlarif/livewire-class-string-bug)

**Reply by joshhanley** [reply]
@fhlarif Thanks so much! Will have a look shortly 🙂

**Comment by joshhanley** [comment]
This issue has been fixed in PR #7579 and tagged in [v3.3.3](https://github.com/livewire/livewire/releases/tag/v3.3.3). Thanks to @fhlarif for the demo repo! 🙂

If you could test and let me know if there are any further issues?

Thanks!

**Reply by fhlarif** [reply]
Hey @joshhanley, thanks for the quick fix! I've upgraded to [v3.3.3](https://github.com/livewire/livewire/releases/tag/v3.3.3) and that solves the issue! Thank you for your hard work!

**Reply by joshhanley** [reply]
@fhlarif no worries! 🙂 Sweet, thanks for confirming

**Comment by marlonbasten** [comment]
Can confirm that it works on my end too now! Thanks for the quick fix.
